package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Jan 22 16:30:06 UTC 2018 - normand@linux.vnet.ibm.com
+
+- in spec file change xsltproc parms to avoid PowerPC build error
+  supposed to solve error since
+  commit#c5e2b30ae60196059134db87e0c462b67d32618f
```

Comment:
can you please reference to bsc#1049297 and increase version, so it will be submitted to jenkins?

Reply:
will do